### ISSUE TITLE
[Bug] A full video audio plays instead of playing the song.

### ISSUE THREAD

**Issue by MichealGun777** [KRTirtho/spotube#251]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

What i mean by this is that lets say that im trying to play a new song by Drake, instead of playing the song it would play a youtube video audio

### Expected Behavior

Heres the video of it.

https://user-images.githubusercontent.com/112316911/193487701-9d34855c-d968-426d-a695-7c124a5943cc.mp4

i found out soon enough the video - https://youtu.be/7WlE25Mfyzg

### Spotube Version

2.4.1

### Operating System

Windows

### Operating System Version

Windows 10

### Installation Source

Official Website (spotube.netlify.app)

**Comment by KRTirtho** [comment]
I guess the song isn't available in YouTube

Try setting the "Popular from author" track match algorithm

**Comment by KRTirtho** [comment]
> Is it trying to get it from youtube instead of spotify?

Yes, of course. Spotify doesn't allow third-party apps to play Music for non-premium users, thus YouTube is used as the Audio provider
